learn/configuration/typo_tolerance.md

```diff
@@ -0,0 +1,64 @@
+# Typo tolerance
+
+Typo tolerance allows users to find documents they're looking for even when they make mistakes while typing. Meilisearch allows you to [configure the typo tolerance feature for each index](/reference/api/typo_tolerance.md#update-typo-tolerance).
+
+:::note
+Meilisearch considers a typo on a query's first character as two typos. This increases performance during search.
+:::
```

Comment:
I think that this deserves more than a note, it is a first-class rule of typo tolerance

Reply:
We use note/warning containers to draw the readers' attention to important information so they don't scroll past it